src/main/java/gift/controller/MemberController.java

```diff
@@ -0,0 +1,75 @@
+package gift.controller;
+
+import gift.service.JwtService;
+import gift.util.LoginMember;
+import gift.domain.Member;
+import gift.service.MemberService;
+import jakarta.validation.Valid;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/members")
+public class MemberController {
+
+    private static final Logger logger = LoggerFactory.getLogger(MemberController.class);
+    private final MemberService memberService;
+    private final JwtService jwtService;
+
+    public MemberController(MemberService memberService, JwtService jwtService) {
+        this.memberService = memberService;
+        this.jwtService = jwtService;
+    }
+
+    @PostMapping("/register")
+    public ResponseEntity<Map<String, String>> register(@Valid @RequestBody Member member) {
+        try {
+            Member savedMember = memberService.createMember(member);
+            Map<String, String> response = new HashMap<>();
+            response.put("token", jwtService.generateToken(savedMember));
+            logger.debug("Register - Generated Token: " + response.get("token"));
```

Comment:
```suggestion
            logger.debug("Register - Generated Token: {}", response.get("token"));
```

이런 방식을 사용해볼 수도 있어요.
다른 곳들도 확인해주세요.

Reply:
반영했습니다. 
불필요한 문자열 결합 연산을 피하여 성능을 향상 시킬 수 있는 법을 확인했습니다.